data/src/main/java/com/bunbeauty/data/repository/CategoryRepository.kt

```diff
@@ -13,13 +14,21 @@ class CategoryRepository @Inject constructor(
     private val categoryMapper: CategoryMapper
 ) : CategoryRepo {
 
-    override suspend fun refreshCategories(token: String, companyUuid: String) {
-        when (val result = networkConnector.getCategoriesByCompanyUuid(token, companyUuid)) {
+    override suspend fun fetchCategories(token: String, companyUuid: String): List<Category> {
```

Comment:
Let's add cache,
`getCategoryList` method should check cache and then go to server
`getCategoryListFromRemote` method should go to server